www/gatsby-node.js

```diff
@@ -105,7 +105,7 @@ exports.createPages = async ({graphql, boundActionCreators}) => {
 
       const createArticlePage = path =>
         createPage({
-          path,
+          path: path,
```

Comment:
Just curious- was this actually necessary?

Reply:
Yes - that is the crazy thing. `fields` was coming up undefined in the graphQL result unless I made this change. 🤷‍♀️

Reply:
Gotcha. I assume this has to do with the version of Node you're using? Not sure which version added support for the object initializer shorthand syntax. Think this was part of ES6/ES2015.

Oh well 😁